src/RapideFix/Validation/ValidatorCollection.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using RapideFix.DataTypes;
+
+namespace RapideFix.Validation
+{
+  public class ValidatorCollection : IValidator
+  {
+    private readonly List<IValidator> _validators;
+
+    public ValidatorCollection(IntegerToFixConverter integerConverter)
+    {
+      _validators = new List<IValidator>();
+      _validators.Add(new FixVersionValidator());
+      _validators.Add(new ChecksumValidator(integerConverter));
+      _validators.Add(new LengthValidator(integerConverter));
+    }
+
+    public bool IsValid(Span<byte> message, FixMessageContext messageContext)
+    {
+      bool result = true;
+      foreach(var validator in _validators)
+      {
+        result &= validator.IsValid(message, messageContext);
```

Comment:
Practically there is no point now to accumulate the bool value. Just return with false if the validation fails and return with true at the end of the method.

Reply:
Good point, I am thinking if we want to collect all validation errors at some point in the future, but I am not sure if there is any value in that. So I will follow up as suggested.